cmd/main.go

```diff
@@ -151,6 +153,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	if withExtensions {
+		// start extension manager
+		extManager, err := extension.NewManager([]string{"myextension"}, "/extensions", log.Log)
+		if err != nil {
+			setupLog.Error(err, "unable to create extension manager")
+			os.Exit(1)
+		}
+		if err = extManager.Start(); err != nil {
+			setupLog.Error(err, "unable to start extension manager")
+			os.Exit(1)
+		}
+	}
+
 	stateOfTheWorld := controllers.NewPolicyMachineryController(mgr, client, log.Log)
```

Comment:
I think we'll eventually need to move this up, and pass that to the `extManager`, I'll work on that as soon as this is merged!

Reply:
Yep 100% was thinking that